src/vs/workbench/contrib/positronNotebook/browser/PositronNotebookInstance.ts

```diff
@@ -510,7 +526,7 @@ export class PositronNotebookInstance extends Disposable implements IPositronNot
 	 * @returns True if the uri is the same as the notebook's uri, false otherwise.
 	 */
 	private _isThisNotebook(uri: URI): boolean {
-		return uri.toString() === this._input.resource.toString();
+		return uri.path === this._input.resource.path;
```

Comment:
Could we use `isEqual` here ([src](https://github.com/posit-dev/positron/blob/3ed030d9c0cf5a7fb59824d290b8a596e99b3c60/src/vs/base/common/resources.ts#L360))?

Reply:
We should. I didn't know that existed! I just checked for methods on the URI class! Thanks!